code/obj/item/paper.dm

```diff
@@ -420,6 +420,50 @@
 		If you want to uninstall a part simply use an empty hand on the maintenance panel. Make sure to close the panel when you are done.<br>
 		In order to use the cargo loader on a crate, simply make ensure the crate is behind the ship, and the loader will handle the rest."}
 
+/obj/item/paper/surplusopsdiary
+	name ="'Journal Entry'"
+	info ={"<br>When they said they said they'd send me to the penal strike force after my incident with the Vega flamethrower and that medic (who totally deserved
+		it for standing four meters behind me when there was a rat nearby), I was expecting a few days of physical labor, not one and a half weeks of living on a damn scrap heap!<br>
+		<br>Thankfully, we've made it to the teleport satellite they said should take us the rest of the way to the research station.
+		Rather, it made itself into our windshield. Luckily, due to my clever thinking and resourceful use of the planks they gave us in case we crashed into anything, I was
+		able to patch it up in a jiffy before we lost most of the air in the cabin. <br>
+		<br> I'm going to wake the other two up now, here's to hoping it'll be an easy job.<br>"}
+
+/obj/item/paper/surplusopsinstructions
+	name ="'Infiltration instructions'"
+
+	info ={"
+		Allrite, numbnuts, since Steve used the instruction papers they gave us for his blunt wrappers, Im going to have to transcribe more or less
+		what they said about why were here and what we need to do <br>
+		<br> 1) First, put the checks into the computer and get yoruself a loadout ammo and utillitys. Different loadouts give different utillitys, and the utillitys can be
+		swapped from one computer to the other, <br>
+		<br> 2) Anything you need, you can probably scavenge around, either on the ship or on the station. Remember to grab tools from the manufactshorer they got us.<br>
+		<br> 2.5) When u are loading up, try to look like your one of them. They do'nt care enough about us to give us any fancey weapons or equipment, so stelth and co-ordinating are going to be the only thing we have going for us. Stick together and stay low.<br>
```

Comment:
```suggestion
		<br> 2.5) When you are loading up, try to look like your one of them. They do'nt care enough about us to give us any fancey weapons or equipment, so stelth and co-ordinating are going to be the only thing we have going for us. Stick together and stay low.<br>
```
reads like textspeak rather than a spelling error